test/integration/custom-networking/custom_networking_test.go

```diff
@@ -106,7 +106,7 @@ var _ = Describe("Custom Networking Test", func() {
 		Context("when connecting to reachable port", func() {
 			BeforeEach(func() {
 				port = customNetworkingSGOpenPort
-				replicaCount = 16
+				replicaCount = 25
```

Comment:
I cannot remember, did we need to update this to 25 or was this just for testing?